tests/conftest.py

```diff
@@ -206,8 +210,76 @@ def expected_catalog_from_db():
          ]}
     ]})
 
+@pytest.fixture
+def expected_catalog_discovered_fixture():
+    return Catalog.from_dict({
+        'streams': [{
+            'database_name': 'FakeDB',
+            'table_name': 'category',
+            'tap_stream_id': 'dev-category',
+            'is_view': False,
+            'stream': 'category',
+            'schema': {
+                'type': 'object',
+                'properties': {
+                    'id': {
+                        'minimum': -2147483648,
+                        'type': 'integer',
+                        'maximum': 2147483647,
+                        'inclusion': 'available'
+                    },
+                    'name': {
+                        'type': 'string',
+                        'inclusion': 'available'
+                    },
+                    'address': {
+                        'type': 'string',
+                        'inclusion': 'unsupported'
+                    }
+                }
+            },
+            'metadata': [
+                {
+                    'breadcrumb': (),
+                    'metadata': {
+                        'selected-by-default': False
+                    }
+                },
+                {
+                    'breadcrumb': (
+                        'properties',
+                        'id'
+                    ),
+                    'metadata': {
+                        'selected-by-default': True,
+                        'sql-datatype': 'int2'
+                    }
+                },
+                {
+                    'breadcrumb': (
+                        'properties',
+                        'name'
+                    ),
+                    'metadata': {
+                        'selected-by-default': True,
+                        'sql-datatype': 'varchar'
+                    }
+                },
+                {
+                    'breadcrumb': (
+                        'properties',
+                        'address'
+                    ),
+                    'metadata': {
+                        'selected-by-default': True,
+                        'sql-datatype': 'varchar'
+                    }
+                }
+            ]
+        }]
+    })
+
 
-@pytest.fixture()
```

Comment:
Idk if an older version of pytest was being used but you cannot call a pytest fixture directly. this fixes the issue.